mscore/editstyle.ui

```diff
@@ -910,39 +910,48 @@
            <bool>false</bool>
           </property>
           <layout class="QGridLayout" name="gridLayout_8">
-           <item row="15" column="2">
-            <widget class="QToolButton" name="resetSystemFrameDistance">
-             <property name="toolTip">
-              <string>Reset to default</string>
+           <item row="19" column="0" colspan="2">
+            <widget class="QCheckBox" name="genCourtesyKeysig">
+             <property name="text">
+              <string>Create courtesy key signatures</string>
              </property>
-             <property name="accessibleName">
-              <string>Reset 'Vertical frame top margin' value</string>
+            </widget>
+           </item>
+           <item row="11" column="5">
+            <widget class="QLabel" name="label_75">
+             <property name="sizePolicy">
+              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+               <horstretch>0</horstretch>
+               <verstretch>0</verstretch>
+              </sizepolicy>
              </property>
              <property name="text">
-              <string notr="true"/>
+              <string>Vertical frame bottom margin:</string>
              </property>
-             <property name="icon">
-              <iconset resource="musescore.qrc">
-               <normaloff>:/data/icons/edit-reset.svg</normaloff>:/data/icons/edit-reset.svg</iconset>
+             <property name="buddy">
+              <cstring>frameSystemDistance</cstring>
              </property>
             </widget>
            </item>
-           <item row="11" column="7">
-            <widget class="QToolButton" name="resetMaxPageFillSpread">
+           <item row="13" column="2">
+            <widget class="QToolButton" name="resetLastSystemFillThreshold">
+             <property name="toolTip">
+              <string>Reset to default</string>
+             </property>
              <property name="accessibleName">
-              <string>Reset &quot;Max. page fill distance&quot; value</string>
+              <string>Reset 'Last system fill threshold' value</string>
              </property>
              <property name="text">
-              <string>...</string>
+              <string notr="true"/>
```

Comment:
lost the ellipsis?

Reply:
I think the git diff makes it appear as though these text fields got swapped, so there's this change. The text it applies to is different

Reply:
Actually it's a reset button, it doesn't even need the ellipsis

Reply:
it may loose that ` notr="true"` then. (a minor nitpick though)

Reply:
I didn't change it, it was done by Qt Designer